experiments/base_experiment.py

```diff
@@ -459,13 +459,15 @@ def train(self):
         patience = 0
 
         # main train loop
+        """
```

Comment:
Why remove this logger output?

Reply:
You cannot call len(self.train_loader) if you build the dataloaders with the SimpleIterDataset class

Reply:
Fair point. It would still be good to have all this information. I think there is a way to implement the len(self.train_loader) thing, namely you have to implement the __len__ method for the SimpleIterDataset properly. A quick fix would be to do something like num_batches = 1239842/batchsize if exp_type=jctagging else len(self.train_loader) where you have to hard-code the number of events in the jetclass dataset.